templates/chatbots/settings_content.html

```diff
@@ -223,27 +250,6 @@ <h4 class="text-md font-bold mb-2">Seed Message</h4>
           });
         }
       })();
-      const voiceProviderTypes = JSON.parse(document.getElementById("voiceProviderTypes").textContent);
-      const voiceOptions = JSON.parse(document.getElementById("voiceOptions").textContent);
```

Comment:
Aren't these still necessary?

Reply:
fix ended up being a bit more involved but confirmed locally it's working: https://github.com/dimagi/open-chat-studio/pull/1578/commits/5be53ba6733cb1934d437c4ea98e7d6c597600d2